README.md

```diff
@@ -62,85 +62,39 @@ Then start the `xremap` daemon by running:
 sudo xremap config.yml
 ```
 
-(You will need to leave it running for your mappings to take effect.)
+To use it without sudo (recommended) run the following commands:
 
-<details>
```

Comment:
If you unfold this, please at least have a separate sub-section, i.e. `###`. Having two distinct options in the same section seems confusing.